rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -355,9 +369,28 @@ def test_migration_11(rotkehlchen_api_server: 'APIServer') -> None:
         'rotkehlchen.data_migrations.manager.MIGRATION_LIST',
         new=[MIGRATION_LIST[5]],
     )
-    with migration_patch:
+
+    rotki.data.db.add_external_service_credentials([ExternalServiceApiCredentials(
+        service=ExternalService.ARBITRUM_ONE_ETHERSCAN,
+        api_key=ApiKey('VQUFYKKJR4RK8HFHYIJ9I93FIUVP44TN99'),
+    )])  # add the arbitrum one etherscan key to the custom db
+    arbitrum_one_addresses = [ethereum_accounts[1]]
+    custom_db_existing_accounts_address = string_to_evm_address('0x2F4c0f60f2116899FA6D4b9d8B979167CE963d25')  # this address has accounts in the custom db  # noqa: E501
+    with ExitStack() as stack:
```

Comment:
This is not a good enough test as it does not seem to test things the same way migraiton 10 test did. Though it's kind of similar.

Check the test of migration 10.In fact abstract the entire thing and use the same exact logic, just add the extra chain for migration 11. I think this would be easier.